apps/nextcloud/vars/vault/main.yml.j2

```diff
@@ -6,3 +6,9 @@
 
 DB_USER: {{ postgresql_credentials.user }}
 DB_PASSWORD: {{ postgresql_credentials.password }}
+
+# nextcloud
+NEXTCLOUD_ADMIN_PASSWORD: {{ lookup('password', '/dev/null length=50') }}
+NEXTCLOUD_PASSWORD_SALT: {{ lookup('password', '/dev/null length=50') }}
+NEXTCLOUD_SECRET: {{ lookup('password', '/dev/null length=50') }}
+INSTANCE_ID: {{ lookup('password', '/dev/null length=10') }}
```

Comment:
Shouldn't it be a human-friendly identifier?

Reply:
This is not what I saw in the documentation https://docs.nextcloud.com/server/latest/admin_manual/configuration_server/config_sample_php_parameters.html#default-parameters